cypress/e2e/Courses/Certificates/FreePlanAwardedCourseCertificate.feature

```diff
@@ -1,20 +1,29 @@
 Feature: Awarded Premium Course Certificates for Free Plan Users
-  Background:
+  Scenario: Free plan users should see the "Download certificate" option for completed and awarded premium courses
     Given a free plan user has some completed awarded premium courses
     And the premium courses page has been navigated to
-
-  Scenario: Free plan users should see the "Download certificate" option for completed and awarded premium courses
-    Given an iOS mobile device is not being used
+    And an iOS mobile device is not being used
     When the user selects the "Completed" tab
     Then the user should see the call to action "Download certificate" for awarded courses
 
   Scenario: Free plan users can successfully download certificates for awarded premium courses
-    Given an iOS mobile device is not being used
+    Given a free plan user has some completed awarded premium courses
+    And the premium courses page has been navigated to
+    And an iOS mobile device is not being used
```

Comment:
We are moving the background inside the tests, so it's easier to intercept api calls